easybuild/easyblocks/p/python.py

```diff
@@ -31,7 +31,7 @@
 @author: Pieter De Baets (Ghent University)
 @author: Jens Timmerman (Ghent University)
 """
-
+import copy
```

Comment:
if this is the main thing we're going to worry about, we should consider EB finished ;)